Makefile

```
@@ -14,33 +17,32 @@ setup-ci-image:
 	docker build -t acorn:v-ci .
 	docker save acorn:v-ci | docker exec -i $$(docker ps | grep k3s | awk '{print $$1}') ctr --address /run/k3s/containerd/containerd.sock images import -
 
-validate:
+GOLANGCI_LINT_VERSION := v1.51.1
+lint:
+	if ! command -v golangci-lint &> /dev/null; then \
+  		echo "Could not find golangci-lint, installing version $(GOLANGLINT_CI_VERSION)."; \
+		curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $$(go env GOPATH)/bin $(GOLANGCI_LINT_VERSION); \
+	fi
```

So the version can be easily overwritten by an environment variable if need be.
```suggestion
lint:
	if ! command -v golangci-lint &> /dev/null; then \
  		echo "Could not find golangci-lint, installing version ${GOLANGLINT_CI_VERSION:-v1.51.1}."; \
		curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $$(go env GOPATH)/bin ${GOLANGCI_LINT_VERSION:-v1.51.1}; \
	fi
```

---

I wasn't able to get this specific change working locally. However, I was able to implement the intent by changing the variable declaration to be a `?=`. Thoughts?